js/gated-content/src/router/index.js

```diff
@@ -9,11 +12,32 @@ const routes = [
     path: '/',
     name: 'Home',
```

Comment:
We were discussing to use mode: hash here

Reply:
It's in use